dev-libs/libtypec/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <longdescription>
+    “libtypec” is aimed to provide a generic interface abstracting all
+    platform complexity for user space to develop tools for efficient
+    USB-C port management. The library can also enable development of
+    diagnostic and debug tools to debug system issues around USB-C/USB
+    PD topology.
+  </longdescription>
+  <maintainer type="person" proxied="yes">
+    <name>Adrian Ratiu</name>
+    <email>adrian.ratiu@collabora.com</email>
+  </maintainer>
+  <maintainer type="project" proxied="proxy">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
+  <upstream>
+    <maintainer type="person">
```

Comment:
Looks like we shouldn't have `type="person"` here when `<maintainer>` is inside `<upstream>`.